Examples/Profiles/Abstract-profile-example.fsh

```diff
@@ -0,0 +1,42 @@
+// @Name: Abstract Profile Example
+// @Description: Abstract profiles can be used to define common constraints used by multiple child profiles. The term "abstract" means that implementers would not create resources conforming to the abstract profile -- instead, they would target conforming to one of non-abstract child profiles. See "Abstract Profile with RuleSets Example" for an alternate approach to this.
+
+Profile:     ExampleAbstractPatientProfile
+Id:          example-abstract-patient-profile
+Parent:      Patient
+Title:       "Example Abstract Patient Profile"
+Description: "Example of an abstract profile of Patient"
+
+// For more information see http://www.hl7.org/fhir/structuredefinition-definitions.html#StructureDefinition.abstract
+* ^abstract = true
+
+// Require at least one value inside the `name` element
+* name 1..*
+
+// Mark elements as MustSupport
+* name and name.given and name.family MS
+
+* deceased[x] 1..1
+
+
+
+// The profiles below will inherit the constraints from the above parent abstract profile
+
+Profile:     ExampleLivingPatientProfile
+Id:          example-living-patient-profile
+Parent:      ExampleAbstractPatientProfile
+Title:       "Example Living Patient Patient Profile"
+Description: "Example of a profile for living patients inheriting from ExampleAbstractPatientProfile"
+
+* deceased[x] only boolean
+* deceasedBoolean = false
+
+
+
```

Comment:
I think our examples should exhibit what we consider to be best practice.  Do we recommend/encourage authors to put a space between the declarations and rules?  And multiple spaces between definitions?  @markkramerus?

Reply:
My preference is to omit blank lines, if that's what you mean.  The exception might be a visual division between major sections of a profile, but even in that case, it would seem better to have a whole-line comment:

//---------------- This section is for setting value sets --------------------

Reply:
I'll remove the extra spacing. 

t's fine to not have whitespace on short profiles like these, but whitespace is important for readability on [longer profiles](https://github.com/HL7/fhir-shc-vaccination-ig/blob/dev/input/fsh/profiles_vaccination.fsh).